frontend/src/app/features/assistance-selection/assistance-selection.component.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { AssistanceLevel } from 'src/app/core/interfaces/assistance-level';
+import { AssistanceSelectionComponent } from './assistance-selection.component';
+
+describe('AssistanceSelectionComponent', () => {
+  let component: AssistanceSelectionComponent;
+  let fixture: ComponentFixture<AssistanceSelectionComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ AssistanceSelectionComponent ]
+    })
+    .compileComponents();
+
+    fixture = TestBed.createComponent(AssistanceSelectionComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+
+  it('handleSelectLevel should change the selected level', () => {
+    const comp = new AssistanceSelectionComponent();
```

Comment:
You shouldn't need to generate a `new AssistanceSelectionComponent();`. There should be a test-wide `component` available.